diff --git a/dist/esm/index.js b/dist/esm/index.js
index c54a90416de4362ebc099688da3bc260c1cd5e21..335f6b3f4fe339e492206cae1455da15134c3660 100644
--- a/dist/esm/index.js
+++ b/dist/esm/index.js
@@ -157,7 +157,7 @@ export function defineConfig(inlineConfig = {}) {
                         TanStackServerFnsPlugin.client,
                         ...(viteConfig.plugins || []),
                         ...(clientViteConfig.plugins || []),
-                        viteReact(opts.react),
+                        // viteReact(opts.react),
                         // TODO: RSCS - enable this
                         // serverComponents.client(),
                     ];
