diff --git a/dist/esm/config/index.js b/dist/esm/config/index.js
index 3af21eca9ec2b9de1ea09daaecf543645bedb95d..f9ec717ebe237cec9b7f882d73cebb7d8e43b609 100644
--- a/dist/esm/config/index.js
+++ b/dist/esm/config/index.js
@@ -120,7 +120,7 @@ export function defineConfig(inlineConfig = {}) {
                         serverFunctions.client({
                             runtime: '@tanstack/start/client-runtime',
                         }),
-                        viteReact(opts.react),
+                        // viteReact(opts.react),
                         // TODO: RSCS - enable this
                         // serverComponents.client(),
                     ];